includes/class-sensei-admin.php

```diff
@@ -634,29 +634,71 @@ private function duplicate_lesson_quizzes( $old_lesson_id, $new_lesson_id ) {
 		}
 	}
 
+	/**
+	 * Sync prerequisite ids after course duplication
+	 *
+	 * @param  array $should_update_prerequisite List with lesson_id and old_prerequisite_id id to update.
+	 * @param  array $duplication_ids_history    History with the id before and after duplication.
+	 * @return void
+	 */
+	private function prerequisite_sync_ids( $should_update_prerequisite, $duplication_ids_history ) {
+		foreach ( $should_update_prerequisite as $lesson_to_update ) {
+			$old_prerequisite_id = $lesson_to_update['old_prerequisite_id'];
+			$new_prerequisite_id = $duplication_ids_history[ $old_prerequisite_id ];
+			add_post_meta( $lesson_to_update['lesson_id'], '_lesson_prerequisite', $new_prerequisite_id );
+		}
+	}
+
+	/**
+	 * Get an update prerequisite object
+	 *
+	 * @param  integer $old_lesson_id ID of the lesson before the duplication.
+	 * @param  integer $new_lesson_id New ID of the lesson.
+	 * @return array                  Object with the id of the lesson to update and its old prerequisite id.
+	 */
+	private function get_update_prerequisite_object( $old_lesson_id, $new_lesson_id ) {
```

Comment:
Hmm. I left a comment here, but it seems to have gotten lost when I submitted my review. It said something like this:

I understand the purpose of this method, we want to "get" an "update prerequisite object". But the second part is confusing at first glance, I think because it starts with a verb. What about calling it a "prerequisite update object" instead? So the method would be `get_prerequisite_update_object`?

Reply:
Looks great! Updating it too!